extensions/mssql/src/mssql.d.ts

```diff
@@ -156,109 +156,22 @@ declare module 'mssql' {
 	export interface DacDeployOptionPropertyBoolean {
 		value: boolean;
 		description: string;
-		displayName: string;
+		propertyName: string;
 	}
 
 	export interface DacDeployOptionPropertyObject {
-		value: SchemaObjectType[];
+		/**
+		 * represents array of numbers, ex: for Include object types options, the selected options are the enum numbers that the current property is holding
```

Comment:
What other things besides the enum values can it represent? Right now it seems very ambiguous from the description what these values might actually be.

Reply:
It holds only the enum values.

Reply:
Please update the comment to make that clear (and indicate what enum values it would contain). And while you're at it I would remove the bit about it being an array of numbers - `number[]` tells us that already. We care about *what* the numbers represent in this case.